src/Core/Search/Filters/AttributeGroupFilters.php

```diff
@@ -45,8 +45,8 @@ public static function getDefaults()
         return [
             'limit' => 50,
             'offset' => 0,
-            'orderBy' => 'position',
-            'sortOrder' => 'desc',
+            'orderBy' => 'id',
```

Comment:
Sorry @mattgoud after discussion with product team (and @paulnoelcholot) it as decided to keep the order by position, but we still need to update this to have the `asc` order by default

The confusion `Done` button will likely be removed in another PR (that will impact all pages relying on this position component)

Reply:
@jolelievre  I removed the rearrange/done button for the attributes/attributes groups pages + put back the filter by position as seen with Amine